backend/src/cache/deployer/deploy-cache-service.sh

```diff
@@ -26,6 +26,18 @@ NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
 MUTATING_WEBHOOK_CONFIGURATION_NAME="cache-webhook-${NAMESPACE}"
 WEBHOOK_SECRET_NAME=webhook-server-tls
 
+# Getting correct kubectl. Kubernetes only supports kubectl versions within +/-1 minor version.
+# kubectl has some resource version information hardcoded, so using too old kubectl can lead to errors
+mkdir -p "$HOME/bin"
+export PATH="$PATH:$HOME/bin"
```

Comment:
You are putting `HOME` after `PATH`, so I think system will still use the default kubectl.
Did I miss anything?

Reply:
Thank you for noticing. You're correct.
I guess it was my "muscle memory" regarding adding things to PATH.
Fixed.